docs/docsite/rst/dev_guide/developing_modules_in_bulk.rst

```diff
@@ -0,0 +1,156 @@
+Partner information for submitting modules
+==========================================
+
+.. contents:: Topics
+
+.. _module_dev_welcome:
+
+Welcome
+```````
+This section discusses some of the specifics around getting multiple related modules into Ansible.
+
+The information in the page is useful for both companies wishing to add modules for their own products, as well as for users of 3rd party products wishing to add Ansible support.
+
+It's based on the knowledge on the Core Team and Community on what we've found is the best practice for the module developer(s), the Community, and the Ansible Core Team.
+
+
+
+Before you start coding
+```````````````````````
+
+Although it's tempting to get straight into coding, there are a few things to be aware of first. This list has been drawn to ensure the best modules are developed and to speed up the review process, which means your modules get accepted into Ansible with minimal fuss.
+
+* Read though all the pages linked off :doc:`developing_modules`, paying particular focus to the :doc:`developing_modules_checklist`.
+* For new modules going into Ansible 2.4 we are raising the bar so they must pass `pep8 --ignore=E402 --max-line-length=160` cleanly.
+* All new modules for Ansible 2.4 must support Python 2.6 and Python 3.5+. If this is an issues mention it in the "Speak to us" section later in this document.
```

Comment:
Suggest: "Starting with Ansible version 2.4, all new modules must support Python 2.6 and Python 3.5+. If this is an issue,  please contact us (see the "Speak to us" section later in this document to learn how)."